ROADMAP.md

```diff
@@ -6,91 +6,44 @@ Community and contributor involvement is vital for successfully implementing all
 
 OpenEBS follows a lean project management approach by splitting the development items into current, near term and future categories.
 
+## Near-term and long-term roadmap (2024 onwards)
+This table contains a list of both near-term and long-term backlog items. Near-term features are currently getting the most active focus and attention within the project. These are prioritized and planned to be completed within the next major release where higher priority is given to usability, stability, resilience, data integrity issues reported by the community. The long-term features require significant design and development efforts, and will be scoped in the future.
+
+OpenEBS follows a release cadence with a new minor release every 3-4 months.
+
+_Note_: The planned release timelines, version numbers and feature priorities are subject to change as the project maintainers/leadership/community continuously update and adjust in response to K8s industry movements, trends and our community influence.
+
+_Note 2_: The table contains the high-level backlog items. For a comprehensive list of features and issues, please refer to https://github.com/openebs/openebs/releases.
+
+| Feature | Description | Local or Replicated | Release timeline | Status |
+| :------ | :---------- | :------------------ | :--------------- | :----- |
+| One installer | Unified Helm installer for all engines, deprecates operator yaml | All | v4.0 (Q1 2024) | Completed |
```

Comment:
nit: One installer -> Single installer

Reply:
How about using the term Unified for Installer, documentation and the plugin.

Reply:
Done